service/rpc2/server.go

```diff
@@ -731,3 +731,26 @@ func (s *RPCServer) ListDynamicLibraries(in ListDynamicLibrariesIn, out *ListDyn
 	out.List = s.debugger.ListDynamicLibraries()
 	return nil
 }
+
+// ExamineMemoryIn holds the arguments of ExamineMemory
+type ExamineMemoryIn struct {
+	Num     int
+	Address uintptr
+}
+
+// ExaminedMemoryOut holds the return values of ExamineMemory
+type ExaminedMemoryOut struct {
+	Address uintptr
```

Comment:
This has to be the same as the input address so it doesn't need to be sent back.

Reply:
Done